src/audits/accessibility/aria-valid-attr.js

```diff
@@ -49,11 +49,10 @@ class ARIAValidAttr extends Audit {
     const rule =
         artifacts.accessibility.violations.find(result => result.id === 'aria-valid-attr');
 
-    return ARIAValidAttr.generateAuditResult(
-      typeof rule === 'undefined',
-      undefined,
-      this.createDebugString(rule)
-    );
+    return ARIAValidAttr.generateAuditResult({
+      value: typeof rule === 'undefined',
```

Comment:
nitty nit: since these are declared so certain to exist, no need to `typeof`, can just test directly against `undefined`